test/extended/fixtures/ldap/ldapserver-deploymentconfig.json

```diff
@@ -58,6 +58,13 @@
             "securityContext": {
               "capabilities": {},
               "privileged": false
+            },
+            "readinessProbe": {
+              "exec": {
+                "command": ["bash", "-c", "\"ldapsearch -x -b dc=example,dc=com | grep organization\""]
```

Comment:
Amazingly this *still* prematurely marks the pod as ready. I have no idea what to do...